pkg/k8sMetadataUtil/k8sMetadataUtil.go

```diff
@@ -66,11 +66,14 @@ func getClientSet() (*kubernetes.Clientset, error) {
 	return clientset, nil
 }
 
+var ClusterUuid k8sTypes.UID = ""
```

Comment:
I'm usually a fan of the functional approach, but in this specific case I prefer this approach